### DISCUSSION TITLE
Cursor position in the audio players

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#853] [Q&A] [answered]
Hi,

I noticed that the cursor position appears in the middle of the track before I press the play button:

![Nltx68Q](https://user-images.githubusercontent.com/83965802/231833407-695dd0e4-8d64-4ff6-ac47-e1c489c1df43.png)

Is it the expected behaviour?

**Comment by ehpersonal38** [comment, marked as answer]
I can't replicate that on Firefox. It must be a weird bug with your browser. But if it's only a cosmetic issue I wouldn't worry about it.

**Reply by blogdemoi** [reply]
I think you are right. I just checked with my smartphone and it displays all right. I updated my Firefox this morning (on Manjaro) and this problem appeared today, so it must be related to the Firefox update...
